planck-cljs/src/planck/repl.cljs

```diff
@@ -54,9 +54,6 @@
        (s/spec-impl ::s/unknown o nil nil)))
     ([o form] (s/spec-impl form o nil nil))))
 
-;; Prefer ES6 Number.isInteger
-(set! integer? (or (.-isInteger js/Number) integer?))
```

Comment:
To be clear, `Number.isInteger` correctly identifies `1.1e22` as an integer. But because we should maintain consistency with ClojureScript (and presumably Clojure where I've just confirmed the bug is also present), we're reverting to the CLJS function, right?

Reply:
Yeah. An argument could be made on the ClojureScript side to perhasps use `Number.isInteger` (but that would make it depend on ES6 _etc_.). In Clojure you end up with floating point coming into the mix, so `integer?` in Clojure has even more slightly different semantics.

Given the complexity of all of this, and given that it was never my intent to make Planck produce different results, this ticket is about reverting things. (It was really only meant to be a perf boost.)

Reply:
Got it—makes sense. Will approve now.